application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -107,6 +97,25 @@ public void onSlashCommand(@NotNull final SlashCommandEvent event) {
             .queue();
     }
 
+    @NotNull
+    private String getLatexToInline(String latex) {
```

Comment:
I think that name could need some improval. More like `convert` instead of `get`. Also, its the opposite direction. inline-latex gets translated into full-latex.